api_coverage.md

```diff
@@ -0,0 +1,108 @@
+| psa_hash_compute                 | :x:                                                                                                                    |
+| psa_hash_compare                 | :x:                                                                                                                    |
+| psa_mac_compute                  | :x:                                                                                                                    |
+| psa_mac_verify                   | :x:                                                                                                                    |
+| psa_cipher_encrypt               | :x:                                                                                                                    |
+| psa_cipher_decrypt               | :x:                                                                                                                    |
+| psa_aead_encrypt                 | :x:                                                                                                                    |
+| psa_aead_decrypt                 | :x:                                                                                                                    |
+| psa_asymmetric_encrypt           | :x:                                                                                                                    |
+| psa_asymmetric_decrypt           | :x:                                                                                                                    |
+| psa_sign_message                 | :x:                                                                                                                    |
+| psa_verify_message               | :x:                                                                                                                    |
+| psa_sign_hash                    | [:white_check_mark:](https://github.com/parallaxsecond/parsec-operations/blob/master/protobuf/sign_hash.proto)         |
+| psa_verify_hash                  | [:white_check_mark:](https://github.com/parallaxsecond/parsec-operations/blob/master/protobuf/verify_hash.proto)       |
+| psa_key_derivation_key_agreement | :x:                                                                                                                    |
+| psa_raw_key_agreement            | :x:                                                                                                                    |
+
+## API support per-provider
+
+This section gives information on the operations and attributes currently supported by each provider. Only support for the operations marked as covered in the table(s) above is considered. Any option marked as supported is necessarily supported for all operations on which it is usable.
+
+### Operation support
+
+| Operation         | Mbed provider      | PKCS 11 provider   | TPM 2.0 provider   |
```

Comment:
I started to rename at various location "Mbed provider" -> "Mbed Crypto provider" as it is more correct